.github/workflows/commitlint.yaml

```diff
@@ -0,0 +1,36 @@
+# Make sure commit subject and description follow the Conventional Commits spec.
+# See https://www.conventionalcommits.org/
+
+name: "commitlint"
+
+on:
+  pull_request: {}
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+permissions:
+  pull-requests: read
+
+jobs:
+  commitlint:
+    runs-on: lab
```

Comment:
This could easily run on GitHub's runners, too.

Reply:
cost saving - everything that can run on our self-hosted runners should go there as we already have a hardware